src/main/webapp/app/tasks/local-task.service.ts

```diff
@@ -22,6 +22,10 @@ export class LocalTaskService implements TaskService {
     return of(task);
   }
 
+  search(search: string): Observable<Task[]> {
+    return of(this.readTasks().filter((task: Task) => task.name.toLowerCase().includes(search)));
```

Comment:
to not use `trim()` was a concious decision?

Reply:
A not conscious decision, I fixed it.